tests/ap_provision.json

```diff
@@ -0,0 +1,13 @@
+{
+  "openconfig-ap-manager:provision-aps": {
+    "provision-ap": [
+      {
+        "mac": "00:11:22:33:44:55",
+        "config": {
+          "country-code": "US",
+          "hostname": "ap1.example.com"
```

Comment:
I think it might be better to keep the hostname same as the remaining tests. "link022-pi-ap", or we can have some more general name.

Reply:
Agreed. Fixed.